index.js

```diff
@@ -779,7 +783,6 @@ var torrentStream = function (link, opts, cb) {
     if (!port) return findPort(opts.port || DEFAULT_PORT, cb)
     engine.port = port
     swarm.listen(engine.port, cb)
-    discovery.updatePort(engine.port)
```

Comment:
I removed this line, it should not be necessary right ?

Reply:
The discovery needs to announce the same port, is that done somewhere else now?